example/proto/demo.pb.custom.preprocess.go

```diff
@@ -0,0 +1,7 @@
+package proto
+
+// CustomPreprocess will be called instead of generated methods
+func (m *Custom) CustomPreprocess() error {
```

Comment:
Maybe we may think of some better name instead of `CustomPreprocess`? What do you think about `MessagePreprocess`?

Reply:
Now we don't generate at all if unused